client/src/components/App/Content/InvestigationForm/TabManagement/InteractionsTab/NewInteractionEventDialog/NewInteractionEventDialog.tsx

```diff
@@ -51,15 +49,15 @@ const NewInteractionEventDialog : React.FC<Props> = (props: Props) : JSX.Element
                 </DialogContent>
             </InteractionEventDialogProvider>
             <DialogActions className={classes.dialogFooter}>
-                <Button 
+                <Button
+                    test-id='cancelNewContactLocation'
                     onClick={() => closeDialog()}
                     color='default' 
                     className={classes.cancelButton}>
                     בטל
                 </Button>
-                <PrimaryButton 
-                    disabled={!canConfirm}
```

Comment:
also this

Reply:
I did not tamper with this line, not sure how it got deleted. This should be restored.

Reply:
Done.